_posts/2019-02-16-ddd-europe.md

```diff
@@ -0,0 +1,340 @@
+
+> "Trying the same thing over and over again expecting different results is the definition of insanity" - Einstein
+
+In fact, Woody said, a constant in his 35 year career was that estimations were always off, and people were always trying to solve this by "getting better at estimates".
+
+Wrong estimates are often not the problem itself, but a symptom of something else.
+It could be that they are off because the requirements were unclear, or that requirements keep changing.
+
+
+### #No estimates
+
+\#No estimates was originally used to refer to reference a blog post Woody had written on a project where they did not use or make estimates.
+But actually 'No estimates' is a placeholder for a larger conversation.
+
+Woody mentions that for some things in life we want estimates, but we never do because we know it's impossible 
+E.g.how long will this clinical trial take?
+How long till we find a cure for cancer?
+How long till you finish this work of art?
+In fact if we have enough data to definitively say how long something will take to develop, we already built it and we don't need to do it again.
+
+
+Next he asked the audience to explain in a single word what an estimate means. 
+Quite some different answers were given, but in general it came down to this list:
+* guess
+* expectation
+* lies
+* misunderstanding
+* approximation
+
+From these answers the following working definition could be extracted:
+> An estimate is a guess of the amount of work time to create a project, a feature or some bit of work in developing software
+
+### Why do we estimate?
+
+Some reasons why we make estimates:
+* planning / budget
+* which approach do we choose / in what order do we do things
+* dependencies on other teams
+
+In software development estimates are often used to attempt to predict the future.
+when will it be done?
+how much will it cost?
+what can we get done this sprint?
+what can we get done for this money?
+
+Basically we use estimates to help us make decisions.
+
+If we have to choose between making project A or project B, we would make an estimation of how long it would take to do either of them.
+But do we really want to choose between project A or B based on a guess?
+Wouldn't it be better to do an MVP of both and see which is working best?
+
+Is on time or on budget a good measure of the results of our decision?
+No, because you cut feature, make it unmaintainable, etc.
+Isn't it better to measure customer satisfaction as a metric for success?
+
+### Conclusion
+After this workshop the audience was left with even more questions.
+But what we did realize is that often people make estimations without any good reason. 
+And sometimes it would be better to reflect on why we do estimations, and see if it really provides us value, and if there is no alternative solution for the problem we're actually trying to solve with estimations.
+
```

Comment:
I would add a "Summing it all" section to summarize you guys' impressions and experiences of the conference in order to end the blogpost.